releasenotes/notes/deprecate-operator-httpfetch.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: feature
+area: installation
+releaseNotes:
+- |
+  **Added** deprecation announcement of fetching charts from arbitrary URLs in `istio-operator`.
```

Deprecated is an acceptable verb, so maybe:
```suggestion
  **Deprecated** fetching charts from arbitrary URLs in `istio-operator`.
```
The deprecation gets lost within the text. This should better highlight it.